notarize_mac_app.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+PKG_DIR=obj/pkg
+
+ASC_USERNAME="$1"
+ASC_PASSWORD="$2"
+BUNDLE_ID="com.cliqz.desktopbrowser"
+BUNDLE_PKG="cliqz.zip"
+
+# create temporary files
+NOTARIZE_APP_LOG=$(mktemp -t notarize-app)
+NOTARIZE_INFO_LOG=$(mktemp -t notarize-info)
+
+# delete temporary files on exit
+function finish {
+	rm "$NOTARIZE_APP_LOG" "$NOTARIZE_INFO_LOG"
+	rm "$BUNDLE_PKG"
+}
+trap finish EXIT
+
+exit 2
```

Comment:
I dont have much idea about bash scripts, but will it not stop script execution here?

Reply:
You are right. Testing stuff, forget to delete. Thank you